Tests/Unit/EmogrifierTest.php

```diff
@@ -692,6 +692,9 @@ public function cssDeclarationWhitespaceDroppingDataProvider()
             'space before trailing semicolon' => [' color:#000 ;', 'color: #000;'],
             'space after trailing semicolon' => [' color:#000; ', 'color: #000;'],
             'space after property value, no trailing semicolon' => [' color:#000; ', 'color: #000;'],
+            'newline before property value, trailing semicolon' => ["\ncolor:#222; ", 'color: #222;'],
+            'newline before colon and after value, trailing semicolon' => ["color\n:#333\n; ", 'color: #333;'],
```

Comment:
The point is to have the tests (or data provider items) as fine-grained as possible so that you know exactly what part of the behavior is broken when something fails.